content/docs/thinking-in-react.md

```diff
@@ -31,116 +31,116 @@ Our JSON API returns some data that looks like this:
+अब जब हमे अपने मॉक कौम्पोनॅन्ट की पहचान हो गई है, उन्हें अब हम पदक्रम में जमाएंगे। मॉक में, जो कौम्पोनॅन्ट दूसरे कौम्पोनॅन्ट के भीतर दिखाई दे रहे है, उन्हें पदक्रम में उनके चाइल्ड की तरह दिखाना होगा:
 
   * `FilterableProductTable`
     * `SearchBar`
     * `ProductTable`
       * `ProductCategoryRow`
       * `ProductRow`
 
-## Step 2: Build A Static Version in React {#step-2-build-a-static-version-in-react}
+## स्टेप 2: React में स्थिर वर्शन बनाए {#step-2-build-a-static-version-in-react}
 
-<p data-height="600" data-theme-id="0" data-slug-hash="BwWzwm" data-default-tab="js" data-user="lacker" data-embed-version="2" class="codepen">See the Pen <a href="https://codepen.io/gaearon/pen/BwWzwm">Thinking In React: Step 2</a> on <a href="https://codepen.io">CodePen</a>.</p>
+<p data-height="600" data-theme-id="0" data-slug-hash="BwWzwm" data-default-tab="js" data-user="lacker" data-embed-version="2" class="codepen">पेन को देखे <a href="https://codepen.io/gaearon/pen/BwWzwm">React में सोचना: स्टेप २</a>, <a href="https://codepen.io">कोडपेन </a>पर।</p>
 <script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
 
-Now that you have your component hierarchy, it's time to implement your app. The easiest way is to build a version that takes your data model and renders the UI but has no interactivity. It's best to decouple these processes because building a static version requires a lot of typing and no thinking, and adding interactivity requires a lot of thinking and not a lot of typing. We'll see why.
+अब जब आपके पास कौम्पोनॅन्ट पदक्रम है, अब समय हो गया है की हम एप्प लागु करे। सबसे आसान तरीका होगा, जहा ऐसा वर्शन बनाए जो आपका डाटा मॉडल लेगा और UI रेंडर करेगा पर उसमे कोई अन्तरक्रियाशीलता नहीं होगी। इन प्रक्रिया को भाग करना श्रेष्ठ रहेगा क्युकी एक स्थिर वर्शन बनाने में लिखना ज्यादा होता है और सोचना कम, और अन्तरक्रियाशीलता जोड़ने में सोचना ज्यादा होता है, और लिखना कम। क्यों, वह हम देखेंगे।
 
-To build a static version of your app that renders your data model, you'll want to build components that reuse other components and pass data using *props*. *props* are a way of passing data from parent to child. If you're familiar with the concept of *state*, **don't use state at all** to build this static version. State is reserved only for interactivity, that is, data that changes over time. Since this is a static version of the app, you don't need it.
+अपनी एप्प का स्थिर वर्शन, जो आपका डाटा मॉडल रेंडर करेगा, बनाने के लिए आपको ऐसे कौम्पोनॅन्टस बनाने होंगे जो दूसरे कौम्पोनॅन्टस को पुनः उपयोग करे, ओर डाटा को *props* से दे। *props* पैरेंट से चाइल्ड को डाटा देने का तरीका है। अगर आप *state* संकल्पना से परिचित है, तो यह स्थिर वर्शन बनाने के लिए *state*  **बिलकुल भी उपयोग न करे**। state सिर्फ अन्तरक्रियाशीलता के लिए आरक्षित है, मतलब, डाटा जो समय के साथ बदलता है. चुकी यह एप्प का स्थिर वर्शन है, आपको इसकी जरुरत नहीं।
```

Comment:
`*state*  **बिलकुल भी` => `*state* **बिलकुल भी`

remove extra space

Reply:
you missed this also

Reply:
removed extra spaces throughout the page